xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -218,11 +218,9 @@
   
  Any callbacks or cancelable operations registered with the <xref:System.Threading.CancellationToken> will be executed.  
   
- Cancelable operations and callbacks registered with the token should not throw exceptions.  
+ It is recommended that cancelable operations and callbacks registered with <xref:System.Threading.CancellationToken> do not throw exceptions. However, this overload of Cancel will aggregate any exceptions thrown into an <xref:System.AggregateException>, such that one callback throwing an exception will not prevent other registered callbacks from being executed.  
   
- However, this overload of Cancel will aggregate any exceptions thrown into an <xref:System.AggregateException>, such that one callback throwing an exception will not prevent other registered callbacks from being executed.  
-  
- The <xref:System.Threading.ExecutionContext> that was captured when each callback was registered will be reestablished when the callback is invoked.  
+ The <xref:System.Threading.ExecutionContext> and, if captured, the <xref:System.Threading.SynchronizationContext>, in use when each callback was registered will be reestablished when the callback is invoked.  
```

Comment:
This latter change should be reverted.  Saying that the SynchronizationContext is "reestablished" suggests it's set as Current; that's not the case.